static/js/settings_bots.js

```diff
@@ -17,6 +17,16 @@ function is_local_part(value, element) {
     return this.optional(element) || /^[\-!#$%&'*+\/=?\^_`{}|~0-9A-Z]+(\.[\-!#$%&'*+\/=?\^_`{}|~0-9A-Z]+)*$/i.test(value);
 }
 
+function type_id_to_string(type_id) {
+    if (type_id === 1) {
+        return "Generic bot";
+    } else if (type_id === 2) {
+        return "Incoming webhook";
+    } else if (type_id === 3) {
+        return "Outgoing webhook";
+    }
```

Comment:
strings in the UI should always be tagged for translation.  See http://zulip.readthedocs.io/en/latest/translating.html for details.